### PR TITLE
Add psg support

### PR DESCRIPTION
**Pull request by hiyer**
Add support for persistent security group in SDK.

### CODE REVIEW

File: qds_sdk/cluster.py [hunk 1]
```diff
@@ -274,6 +274,12 @@ def _parse_create_update(cls, args, action):
                                     help="location for ssh key to use to" +
                                          " login to the instance")
 
+        security_group.add_argument("--persistent-security-groups",
+                                    dest="persistent_security_groups",
+                                    help="a security group to associate with each" +
+                                         " node of the cluster. Typically used" +
+                                         " to provide access to external hosts")
+
```
Comment (mindprince) [inline qds_sdk/cluster.py]: The help says `a security group...` but the argument name is `...groups`. Which one is correct? If the plural one is correct - let's also provide example usage in the help message.

Comment (hiyer) [inline qds_sdk/cluster.py]: The input name is groups, but we support only one group at present. It's
been on my mind as well - maybe I'll change the name to
persistent_security_group itself.

**Comment by vrajat** [comment]
Can this go in now to master ?

**Comment by mindprince** [comment]
@hiyer Forget to ask before, can we add a test case for this. Thanks!

**Comment by hiyer** [comment]
Sure, will do.

**Comment by vrajat** [comment]
Ping. Please Please pretty please ?
PSG is very important for HBase. Also I am blocked on submitting a pyapi test that checks connectivity between Hive and HBase. Both clusters have to be in PSG for that. Can I get an ETA ?

**Comment by hiyer** [comment]
Unit test has been added.

**Comment by mindprince** [comment]
squash-merged to master and cherry-picked to unreleased.

**Pull request closed without merging**